TOC-tidb-cloud.md

```diff
@@ -114,6 +114,8 @@
         - [Naming Conventions for Data Import](/tidb-cloud/naming-conventions-for-data-import.md)
         - [Troubleshoot Access Denied Errors during Data Import from Amazon S3](/tidb-cloud/troubleshoot-import-access-denied-error.md)
   - [Export Data from TiDB](/tidb-cloud/export-data-from-tidb-cloud.md)
+- Streaming your TiDB data
+  - [To MySQL sink](/tidb-cloud/changefeed-sink-to-mysql.md)
```

Comment:
Why not add `tidb-cloud/changefeed-overview.md` and `tidb-cloud/changefeed-sink-to-apache-kafka.md` in TOC?

Reply:
Kafka sink are still not GA

Reply:
But we plan to public kafka sink

Reply:
Then please add Kafka sink doc in TOC.